dev/start.bat

```diff
@@ -72,13 +72,21 @@
 @ECHO.
 @ECHO Note: This choice can be overridden by setting the 'installer_type' key in ~\.condarc.
 @ECHO.
+@CALL :RESET_ERRORLEVEL
 @SET /P "_INSTALLER_TYPE=Enter choice [1]: "
-@IF NOT %ErrorLevel%==0 @EXIT /B 1
+@SET "_INSTALLER_TYPE_PROMPT=%ErrorLevel%"
+@SET "_INSTALLER_TYPE_RAW=%_INSTALLER_TYPE%"
 :: normalize user input
 @IF "%_INSTALLER_TYPE%"=="1" @SET "_INSTALLER_TYPE=miniconda"
 @IF "%_INSTALLER_TYPE%"=="" @SET "_INSTALLER_TYPE=miniconda"
 @IF "%_INSTALLER_TYPE%"=="2" @SET "_INSTALLER_TYPE=miniforge"
-@IF "%_INSTALLER_TYPE%"!="miniconda" @IF "%_INSTALLER_TYPE%"!="miniforge" (
+@IF NOT "%_INSTALLER_TYPE_PROMPT%"=="0" (
+    @IF NOT "%_INSTALLER_TYPE_RAW%"=="" (
+        @ECHO Error: failed to read installer selection 1>&2
+        @EXIT /B 1
+    )
+)
```

Comment:
Since there is no way to distinguish between an `Enter` and `Ctrl-C` (both result in `%_INSTALLER_TYPE%==""` and `%ErrorLevel%==1`) this extra check doesn't help us:

```suggestion
```